EXPRESSIONS.md

```diff
@@ -0,0 +1,80 @@
+# Example WDL for CancerExomePipeline_v2
+
+```
+task M2 {
+  File ref_fasta
+  File ref_fasta_dict
+  File ref_fasta_fai
+  File tumor_bam
+  File tumor_bai
+  File normal_bam
+  File normal_bai
+  File intervals
+  String m2_output_vcf_name
+
+  command {
+    java -jar /task/GenomeAnalysisTK_latest_unstable.jar -T M2 \
+    --no_cmdline_in_header -dt NONE -ip 50 \
+    -R ${ref_fasta} \
+    -I:tumor ${tumor_bam} \
+    -I:normal ${normal_bam} \
+    -L ${intervals} \
+    -o ${m2_output_vcf_name} \
+    --disable_auto_index_creation_and_locking_when_reading_rods
+  }
+  runtime {
+    docker: "gcr.io/broad-dsde-dev/cep3"
+  }
+  output {
+    File m2_output_vcf = "${m2_output_vcf_name}"
+  }
+}
+
+workflow CancerExomePipeline_v2 {
+  call M2
+}
+```
```

Comment:
what's the WDL got to do with anything? i feel like including it risks confusing WDL input/output variables with rawls expressions.